python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py

```diff
@@ -104,6 +105,10 @@ def all_reduce_min(self, tensor, *args, **kwargs):
         return dist.all_reduce(tensor, *args,
                                **all_reduce_min_kwargs)
 
+class RunnerMode(Enum):
```

Comment:
Does this PR conflict with https://github.com/intel-analytics/BigDL/pull/7050 ?
Or at least RunnerMode could be combined if we decide to add this

Reply:
> Does this PR conflict with #7050 ? Or at least RunnerMode could be combined if we decide to add this

I had synced with Ge about this before, and I think it's beneficial to set a flag like this to indicate runner status.

Reply:
Is this necessary? A runner may have multiple modes right?

Reply:
> Is this necessary? A runner may have multiple modes after its creation right?

No, so using string comparision instead?

Reply:
synced offline with ge.